src/plugins/event_annotation_listing/public/components/group_editor_flyout/group_editor_controls/group_editor_controls.tsx

```diff
@@ -166,46 +156,92 @@ export const GroupEditorControls = ({
-                  value,
-                  text: name ?? title,
-                }))}
-                value={group.indexPatternId}
-                onChange={({ target: { value } }) =>
-                  update({
-                    ...group,
-                    indexPatternId: value,
-                    dataViewSpec:
-                      value === adHocDataView?.id ? adHocDataView.toSpec(false) : undefined,
-                  })
-                }
-              />
-            </EuiFormRow>
-            <EuiFormRow
-              label={i18n.translate('eventAnnotationComponents.groupEditor.addAnnotation', {
-                defaultMessage: 'Annotations',
-              })}
-            >
-              <AnnotationList
-                annotations={group.annotations}
-                selectAnnotation={setSelectedAnnotation}
-                update={(newAnnotations) => update({ ...group, annotations: newAnnotations })}
-              />
-            </EuiFormRow>
-          </>
-        )}
+        <EuiFormRow
+          label={i18n.translate('eventAnnotationListing.groupEditor.dataView', {
+            defaultMessage: 'Data view',
+          })}
+          isInvalid={!isDataViewValid(currentDataView)}
+          error={
+            !isDataViewValid(currentDataView)
+              ? i18n.translate('eventAnnotationListing.groupEditor.dataViewMissingError', {
+                  defaultMessage: 'The previously selected data view no longer exists.',
+                })
+              : ''
+          }
+        >
+          <EuiSelect
+            compressed
+            data-test-subj="annotationDataViewSelection"
+            isInvalid={!isDataViewValid(currentDataView)}
+            options={dataViews.map(({ id: value, title, name }) => ({
+              value,
+              text: name ?? title,
+            }))}
+            value={isDataViewValid(currentDataView) ? group.indexPatternId : undefined}
+            hasNoInitialSelection={true}
+            onChange={({ target: { value } }) => {
+              const selectedDataView = dataViews.find(({ id }) => id === value);
+
+              if (!selectedDataView?.id) {
+                return;
+              }
+
+              update({
+                ...group,
+                indexPatternId: value,
+                dataViewSpec: isAdHocDataView(selectedDataView.id)
+                  ? selectedDataView.toSpec(false)
+                  : undefined,
+              });
+            }}
+          />
+        </EuiFormRow>
       </EuiForm>
+      <div
+        css={css`
+          margin-top: ${euiThemeVars.euiSize};
+          padding-top: ${euiThemeVars.euiSize};
+          position: relative;
+
+          &:before {
+            content: '';
+            position: absolute;
+            top: 0;
+            right: -${euiThemeVars.euiSize};
+            left: -${euiThemeVars.euiSize};
+            border-top: 1px solid ${euiThemeVars.euiColorLightShade};
+          }
+        `}
+      >
```

Comment:
Thanks for bringing this up. I prefer this way because 
1. it doesn't force me to override EUI styles
2. it's what is already done with the rest of the annotations controls